tasks/section1.yml

```diff
@@ -1180,7 +1180,7 @@ print $NF }'"
       - rule_1.6.2
 
 - name: "SCORED | 1.7.1.1 | AUDIT | Ensure message of the day is configured properly"
-  command: /bin/true
+  command: egrep '(\\v|\\r|\\m|\\s)' /etc/motd
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*